src/main/webapp/app/overview/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.scss

```diff
@@ -0,0 +1,85 @@
+.container {
+    position: relative;
+}
+
+.chat-widget {
+    position: fixed;
+    bottom: 100px;
+    right: 60px;
+    width: 300px;
+    height: 400px;
+    background-color: white;
+    box-shadow: 0 0 50px rgba(0, 0, 0, 0.4);
+}
+
+.chat-header {
+    display: flex;
+    justify-content: space-between;
+    height: 50px;
+    background-color: #212529;
+    color: white;
+    font-weight: bold;
+    padding: 10px;
+}
+
+.header-icon {
+    border: none;
+    background: transparent;
+    color: grey;
+}
+
+.chat-body {
+    height: calc(100% - 100px);
+    overflow-y: scroll;
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+}
+
+.client-chat {
+    width: fit-content;
+    background-color: #f2f2f2;
+    color: dimgrey;
+    padding: 10px;
+    border-radius: 10px;
+    margin-bottom: 10px;
+}
+
+.my-chat {
+    width: fit-content;
+    margin-left: auto;
+    background-color: #b7dfb9;
+    color: dimgrey;
+    padding: 10px;
+    border-radius: 10px;
+    margin-bottom: 10px;
+}
+
+.chat-input {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 10px;
+    background-color: #f2f2f2;
+}
+
+.chat-input input {
+    flex: 1;
+    margin-right: 10px;
+    padding: 5px 15px;
+    border: 1px solid lightgray;
+    border-radius: 20px;
+}
+
+.btn-circle {
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    background-color: #4caf50;
+    border: none;
+}
+
+.btn-circle:hover,
+.btn-circle:active {
+    background-color: #3e913e;
```

Comment:
Please do not hardcode colors. Use predefined default colors instead; only specify your own colors if strictly necessary. Please follow the guidelines on client theming: https://docs.artemis.cit.tum.de/dev/guidelines/client-design/

Reply:
Done